dhis-2/build-dev.sh

```diff
@@ -51,8 +51,8 @@ cp -f "$DIR/dhis-web/dhis-web-portal/target/dhis.war" "$ARTIFACTS/dhis.war"
 print "Build succeeded, creating Docker image $IMAGE:$TAG..."
 
 cd $ARTIFACTS
-sha256sum ./dhis.war > ./sha256sum.txt
```

Comment:
these 2 removals are probably not intentional

Reply:
It's just a small formatting fix